return table.freeze {
    useCamera = require("@self/useCamera"),
    useViewport = require("@self/useViewport"),
    useEventListener = require("@self/useEventListener"),
    useTimer = require("@self/useTimer"),
    useMouse = require("@self/useMouse"),
    useTagged = require("@self/useTagged"),
    useInterval = require("@self/useInterval"),
    useDeferCallback = require("@self/util/deferCallback"),
    useDeferFlec = require("@self/useDeferFlec"),
    useDeferEffect = require("@self/useDeferEffect"),
    useDebounceCallback = require("@self/useDebounceCallback"),
    useDebounceFlec = require("@self/useDebounceFlec"),
    useDebounceEffect = require("@self/useDebounceEffect"),
    useThrottleCallback = require("@self/useThrottleCallback"),
    useThrottleFlec = require("@self/useThrottleFlec"),
    useThrottleEffect = require("@self/useThrottleEffect"),
}